doc/book/admin/monitoring/metrics_reference.rst

```diff
@@ -0,0 +1,1012 @@
+                This tuples are no longer in the index, but MVCC does not allow them to be removed.
+                This metric always has the label ``{kind="..."}``,
+                which has the following possible values:
+
+                *   ``count``
+                    Number of retained ``read_view`` tuples / number of stories.
+                *   ``total``
+                    Amount of bytes used by retained ``read_view`` tuples.
+
+        *   -   ``tnt_memtx_mvcc_tuples_tracking_stories``
+            -   Tuples that are not directly used by any transactions, but are used by MVCC to track reads.
+                This metric always has the label ``{kind="..."}``,
+                which has the following possible values:
+
+                *   ``count``
+                    Number of ``tracking`` tuples / number of tracking stories.
+                *   ``total``
+                    Amount of bytes used by stories ``tracking`` tuples.
+
+        *   -   ``tnt_memtx_mvcc_tuples_tracking_retained``
+            -   Tuples that are not directly used by any transactions, but are used by MVCC to track reads.
+                This tuples are no longer in the index, but MVCC does not allow them to be removed.
+                This metric always has the label ``{kind="..."}``,
+                which has the following possible values:
+
+                *   ``count``
+                    Number of retained ``tracking`` tuples / number of stories.
+                *   ``total``
+                    Amount of bytes used by retained ``tracking`` tuples.
+
+
+..  _metrics-reference-read-view:
+
+~~~~~~~~~~~~~~~~~~~~
+Read view statistics
+~~~~~~~~~~~~~~~~~~~~
+
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tuples_data_total``
+            -   Total amount of memory (in bytes) allocated for data tuples.
+                This includes ``tnt_memtx_tuples_data_read_view`` and
+                ``tnt_memtx_tuples_data_garbage`` metric values plus tuples that
+                are actually stored in memtx spaces.
+
+        *   -   ``tnt_memtx_tuples_data_read_view``
+            -   Memory (in bytes) held for read views.
+
+        *   -   ``tnt_memtx_tuples_data_garbage``
+            -   Memory (in bytes) that is unused and scheduled to be freed
+                (freed lazily on memory allocation).
+
+        *   -   ``tnt_memtx_index_total``
+            -   Total amount of memory (in bytes) allocated for indexing data.
+                This includes ``tnt_memtx_index_read_view`` metric value
+                plus memory used for indexing tuples
+                that are actually stored in memtx spaces.
+
+        *   -   ``tnt_memtx_index_read_view``
+            -   Memory (in bytes) held for read views.
+
+
+..  _metrics-reference-tarantool-config:
+
+Tarantool configuration
+-----------------------
+
+**Since:** :doc:`3.0.0 </release/3.0.0>`.
```

Comment:
Well, if we treat `metrics` as a built-in package, these two are expected to be "since 3.1.1" (https://github.com/tarantool/tarantool/pull/10220)

Reply:
Created a doc issue for this: https://github.com/tarantool/doc/issues/4365

Reply:
And what's about this version of doc?